xtask/codegen/src/formatter.rs

```diff
@@ -642,25 +674,13 @@ impl NodeModuleInformation {
     }
 }
 
-/// Convert an AstNode name to a path / Rust module name
-fn name_to_module(kind: &NodeKind, in_name: &str, language: LanguageKind) -> NodeModuleInformation {
```

Comment:
This didn't actually change, the function just moved below the new `get_node_concept` function.